src/jvm/test/resources/expected_logs/multiple_suspension_points_channels_trace_debugger.txt

```diff
@@ -53,14 +53,14 @@ The following interleaving leads to the error:
 |                                                                                 |     switch (reason: coroutine is suspended)                                                        |
 |   counter = 1                                                                   |                                                                                                    |
 |   channel2.receive() [suspendable: Continuation#4]: 1                           |                                                                                                    |
-|   1.intValue(): 1                                                               |                                                                                                    |
+|   intValue(): 1                                                                 |                                                                                                    |
 |   Boxing.boxInt(2): 2                                                           |                                                                                                    |
 |   channel1.send(2) [suspendable: Continuation#4]: COROUTINE_SUSPENDED           |                                                                                                    |
 |     send$suspendImpl(BufferedChannel#1, 2, Continuation#4): COROUTINE_SUSPENDED |                                                                                                    |
 |     switch (reason: coroutine is suspended)                                     |                                                                                                    |
 |                                                                                 |   ResultKt.throwOnFailure(Unit)                                                                    |
 |                                                                                 |   channel1.receive() [suspendable: Continuation#1]: 2                                              |
-|                                                                                 |   2.intValue(): 2                                                                                  |
+|                                                                                 |   intValue(): 2                                                                                    |
```

Comment:
1. We should not show function calls for boxing/unboxing
2. It is called not on `this`, so there should be a bug

Reply:
It's something added by the compiler, I suppose, and is called on an Integer object.